apps/ehr/src/pages/reports/AiAssistedEncounters.tsx

```diff
@@ -92,14 +122,76 @@ const useAiAssistedEncounters = (
       // Build location filter if a location is selected
       const locationIds = selectedLocationId !== 'all' ? [selectedLocationId] : undefined;
 
-      // Call the Zambda function to get AI-assisted encounters
-      const response = await getAiAssistedEncountersReport(oystehrZambda, {
-        dateRange: { start, end },
-        locationIds,
+      // Calculate the date range in days
+      const startDate = DateTime.fromISO(start);
+      const endDate = DateTime.fromISO(end);
+      const daysDifference = endDate.diff(startDate, 'days').days;
+
+      console.log(`Date range is ${daysDifference.toFixed(2)} days`);
+
+      // If the date range is <= 5 days, make a single request
+      if (daysDifference <= 5) {
```

Comment:
use a const somewhere for the batch days in case we want to change it in the future